docs/community/user-guides/multi-tenant-deployment-with-powershell.md

```diff
@@ -0,0 +1,185 @@
+}
+```
+
+Run the following commands. Note that the _security_ command must come before
+the _admin_ command.
+
+```powershell
+Reset-MinimalTemplateDatabase
+Reset-PopulatedTemplate
+Reset-SecurityDatabase
+Reset-AdminDatabase -DropDatabase:$false
+```
+
+Now edit the user secret file, change the tenant name in the connection strings,
+and rerun the commands above.
+
+## Configure ODS Instances
+
+Next, need to setup the `dbo.OdsInstances` table in each `EdFi_Tenant{?}_Security` database.
+
+```sql
+DECLARE @OdsInstanceName nvarchar(100) = 'Sandbox'
+DECLARE @OdsInstanceType nvarchar(100) = 'Sandbox'
+DECLARE @OdsInstanceConnectionString nvarchar(500) =
+ 'server=(local);trusted_connection=True;database=EdFi_Tenant1_{0};application name=EdFi.Ods.WebApi;Encrypt=False'
+
+DECLARE @OdsInstanceId int
+
+-- Ensure OdsInstance exists
+SELECT @OdsInstanceId = OdsInstanceId FROM [dbo].[OdsInstances]
+ WHERE [Name] = @OdsInstanceName and InstanceType = @OdsInstanceType
+
+IF(@OdsInstanceId IS NULL)
+BEGIN
+    INSERT INTO [dbo].[OdsInstances] ([Name], InstanceType, ConnectionString)
+    VALUES (@OdsInstanceName, @OdsInstanceType, @OdsInstanceConnectionString)
+    SET @OdsInstanceId = SCOPE_IDENTITY()
+END
+ELSE
+BEGIN
+    UPDATE [dbo].[OdsInstances] SET ConnectionString = @OdsInstanceConnectionString WHERE OdsInstanceId = @OdsInstanceId
+END
+```
+
+## Configure for Multi-Tenancy
+
+Finally, update the tenancy-settings in user settings.
+
+```json
+{
+    "ApiSettings": {
+        "Features": [
+            {
+                "IsEnabled": true,
+                "Name": "MultiTenancy"
+            }
+        ]
+    },
+    "Tenants": {
+        "Tenant1": {
+            "ConnectionStrings": {
+                "EdFi_Admin": "Server=(local); Database=EdFi_Tenant1_Admin; Trusted_Connection=True; Application Name=EdFi.Ods.WebApi;",
+                "EdFi_Security": "Server=(local); Database=EdFi_Tenant1_Security; Trusted_Connection=True; Persist Security Info=True; Application Name=EdFi.Ods.WebApi;"
+            }
+        },
+        "Tenant2": {
+            "ConnectionStrings": {
+                "EdFi_Admin": "Server=(local); Database=EdFi_Tenant2_Admin; Trusted_Connection=True; Application Name=EdFi.Ods.WebApi;",
+                "EdFi_Security": "Server=(local); Database=EdFi_Tenant2_Security; Trusted_Connection=True; Persist Security Info=True; Application Name=EdFi.Ods.WebApi;"
+            }
+        }
+    }
+}
+```
+
+:::warning
+
+TODO: use correct installtype. Configure year-based routing.
+
+:::
+
+## Bonus: Combining Admin and Security Databases
+
+To lower the maintenance and cost burden, you can deploy the `EdFi_Admin` and
+`EdFi_Security` databases to the _same database_, so long as you first deploy
+the Security database, and then deploy the Admin database _with `dropdatabases`
+disabled_ as shown in the script above. Then, simply use the same database name
+in both the `EdFi_Admin` and `EdFi_Security` connection strings.
```

Comment:
Need to finish the idea in this paragraph... and use the same scripts / steps shown above to manage the deployment.